libs/cdk/src/lib/utils/directives/focusable-item/focusable-item.directive.ts

```diff
@@ -127,8 +149,8 @@ export class FocusableItemDirective implements HasElementRef {
     }
 
     /** @hidden */
-    @HostListener('focusin')
-    async _onFocusin(): Promise<void> {
+    // @HostListener('focusin')
```

Comment:
If we leave these comments maybe make a note that it's really handled in `runOutsideAngular`